eng/versioning/version_client.txt

```diff
@@ -74,23 +74,23 @@ com.azure:azure-communication-jobrouter;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-rooms;1.0.1;1.1.0-beta.1
 com.azure:azure-containers-containerregistry;1.1.1;1.2.0-beta.1
 com.azure:azure-containers-containerregistry-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-core;1.40.0;1.41.0
-com.azure:azure-core-amqp;2.8.6;2.9.0-beta.3
+com.azure:azure-core;1.41.0;1.42.0-beta.1
+com.azure:azure-core-amqp;2.8.6;2.9.0-beta.4
```

Comment:
I guess we didn't release azure-core-amqp beta.3 right? wondering why the version bump?

Reply:
We did release beta 3, see [here](https://central.sonatype.com/artifact/com.azure/azure-core-amqp/2.9.0-beta.3).